packages/sanity/src/core/studio/components/navbar/search/components/filters/filter/inputs/asset/AssetSourceError.tsx

```diff
@@ -12,13 +15,13 @@ export function AssetSourceError(props: ResponsivePaddingProps) {
         </TextWithTone>
         <Stack space={4}>
           <TextWithTone size={1} tone="caution" weight="semibold">
-            No valid asset sources found.
+            {t('navbar.search.error.no-valid-asset-source-title')}
           </TextWithTone>
           <TextWithTone size={1} tone="caution">
-            Currently, only the default asset source is supported.
+            {t('navbar.search.error.no-valid-asset-source-only-default-description')}
           </TextWithTone>
           <TextWithTone size={1} tone="caution">
-            Please ensure it's enabled in your studio configuration file.
+            {t('navbar.search.error.no-valid-asset-source-check-config-description')}
           </TextWithTone>
```

Comment:
**question:** could this also be better served by `<Trans>`? 

Which could potentially allow breaking this up into two tokens instead of three: `navbar.search.error.no-valid-asset-source-title` and `navbar.search.error.no-valid-asset-source-description`

Reply:
Usually the Trans component is to be used when you have variables that require different react components to be used within. 
I am not sure what would be the benefit in this case 🤔

Reply:
(To summarise what we spoke about separately)

This is more an issue rooted in the original source markup, which has led to having two keys which are essentially 'description-line-1' and 'description-line-2'. 

We agreed that we should try and avoid similar markup in future, ideally representing this as the folllowing (and allow text to line break naturally):
```
<TextWithTone size={1} tone="caution">
  {t('navbar.search.error.no-valid-asset-source-only-description')}
</TextWithTone>
```